File: 4.Function/Lecture 21/Program6.java

import java.io.*;
class Demo{
	

void Demo(){

		System.out.println("Constructor");
}

Demo(){

	System.out.println(this);
	System.out.println("cons");
}

public static void main(String[] args){
	
	Demo obj = new Demo();
	System.out.println(obj);

	obj.Demo();

}

}

File: 5.Constructor/Lecture 23/Program6.dart


class Demo{
	
	Demo(){

		print(this);

		print(this.hashCode);

		print(identityHashCode(this));

		print("Constructor End");
	}
}

void main(){
	
	Demo obj = new Demo();

	print(obj);

	print(obj.hashCode);

	print(identityHashCode(obj));

}